src/custom-components/CustomButton.tsx

```diff
@@ -0,0 +1,109 @@
+import { ReactNode, FC, useState } from "react";
```

Comment:
My main question with this entire component: Why?
What benefit does this provide over normal valve buttons?